### DISCUSSION TITLE
Jump to the target of a symlink?

### DISCUSSION THREAD

**Discussion by magical-heyrovsky** [jarun/nnn#1792] [Q&A] [answered]
As far as I can tell it's not possible to jump to the target of a symlink and there is also no plugin for it, correct? If that's the case I will create a plugin for it because I could use that functionality on a daily basis.

**Comment by jarun** [comment]
Currently there is no such option. We can have a plugin to jump to the target of the current directory.

**Reply by magical-heyrovsky** [reply, marked as answer]
As you said, there is no plugin, BUT I just found this in the examples:

https://github.com/jarun/nnn/tree/master/plugins#change-directory-to-the-location-of-a-link-using-helper-script-with-specific-context-current

This is exactly what I was looking for and I use it as a plugin now.
